src/main/java/com/karankumar/bookproject/backend/repository/CustomShelfRepository.java

```diff
@@ -19,10 +19,12 @@
 package com.karankumar.bookproject.backend.repository;
 
 import com.karankumar.bookproject.backend.entity.CustomShelf;
+import com.karankumar.bookproject.backend.entity.account.User;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import java.util.List;
 
 public interface CustomShelfRepository extends JpaRepository<CustomShelf, Long> {
-    List<CustomShelf> findByShelfName(String shelfName);
+    List<CustomShelf> findAllByUser(User user);
```

Comment:
Do we really need this method? What would be the use case for this method?

Reply:
this method is used instead of `findAll()`

Reply:
@amit1307 I think @Burhak is right. This will allow us to get all of the custom shelves that belong to a specific user